osx/release-docs/README.md

```diff
@@ -5,13 +5,15 @@ This is the [KA Lite](https://github.com/learningequality/ka-lite/) application
 
 It is used to control and monitor the [KA Lite](https://github.com/learningequality/ka-lite/) server by [Foundation for Learning Equality](https://learningequality.org/).
 
-It uses [PyRun](http://www.egenix.com/products/python/PyRun/) to isolate the KA Lite environment from your system's [Python](https://www.python.org/) application.
+It uses [PEX](https://pex.readthedocs.io/en/stable/) to create `kalite.pex` executable.
 
 
 ## Install KA Lite
 
-1. Download the [KA Lite Mac OS X Installer for 0.16](http://pantry.learningequality.org/downloads/ka-lite/0.16/installers/mac/).
-1. Double-click the downloaded `KA-Lite.pkg` package and click "Continue" when prompted about the "This package will run a program to determine if the software can be installed." message.
+1. Download the [KA Lite Mac OS X Installer for 0.17](http://pantry.learningequality.org/downloads/ka-lite/0.17/installers/mac/).
+1. Double-click the downloaded `KA-lite-Installer.dmg`.
+1. KA Lite Mac OS X Installer required Python 2.7.12 or higher versions. If you're not sure the version of python installed in your machine, double-click README.md for the instructions.
```

Comment:
Please re-phrase to `... If you're not sure which version of Python is installed in your machine...`?